include/spirv/unified1/spirv.core.grammar.json

```
@@ -17735,6 +17747,13 @@
           "extensions" : [ "SPV_INTEL_ternary_bitwise_function"],
           "version" : "None"
         },
+        {
+          "enumerant" : "UntypedVariableLengthArrayINTEL",
+          "value" : 6244,
```

In the spec PR https://github.com/KhronosGroup/SPIRV-Registry/pull/352 this capability has value 6243, not 6244.

```suggestion
          "value" : 6243,
```